### PR TITLE
Rana scanner flash

### PR DESCRIPTION
**Pull request by sunjogthind**
Build Gradle issue resolved now.
Working flash.

### CODE REVIEW

File: app/src/main/java/com/example/qrchive/Classes/MapModel.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 public class MapModel {
 
-    private final double radius = 1000; // 1000 m raidus
+    private final double radius = 1000; // 1000 m radius
```
Comment (jsh3lly) [inline app/src/main/java/com/example/qrchive/Classes/MapModel.java]: Don't commit this, this is deleted in a different PR anyway

File: .idea/deploymentTargetDropDown.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+          <Key>
+            <type value="SERIAL_NUMBER" />
+            <value value="1311c459" />
+          </Key>
+        </deviceKey>
+      </Target>
+    </runningDeviceTargetSelectedWithDropDown>
+    <timeTargetWasSelectedWithDropDown value="2023-03-11T20:21:28.648567Z" />
+  </component>
+</project>
```
Comment (jsh3lly) [inline .idea/deploymentTargetDropDown.xml]: Don;t commit this

**Review by jsh3lly — COMMENTED** [review]
Apart from the comments, LGTM

File: .idea/deploymentTargetDropDown.xml (hunk 2 above)
Comment (sunjogthind) [reply .idea/deploymentTargetDropDown.xml]: Will this cause an issue?

File: app/src/main/java/com/example/qrchive/Classes/MapModel.java (hunk 1 above)
Comment (sunjogthind) [reply app/src/main/java/com/example/qrchive/Classes/MapModel.java]: Cool

File: .idea/deploymentTargetDropDown.xml (hunk 2 above)
Comment (sunjogthind) [reply .idea/deploymentTargetDropDown.xml]: removed the folder

File: app/src/main/java/com/example/qrchive/Classes/MapModel.java (hunk 1 above)
Comment (sunjogthind) [reply app/src/main/java/com/example/qrchive/Classes/MapModel.java]: fixed -hd

**Review by jsh3lly — CHANGES_REQUESTED** [review]
DO NOT delete the entire `.idea/` folder and then push it to Github. That means that when we try to pull origin, OUR `.idea/` folder gets deleted.
What you rather had to do was rollback ur commit (or reset, whatever), and then **unstage** the file u staged (which was [.idea/deploymentTargetDropDown.xml](https://github.com/CMPUT301W23T08/qrchive/pull/74/files/5860ef66c74dbd3cf6899b06a2bc6db865d19711#diff-214b4b51f1f72c46bb9980750ced0cdb7ac8dd53435e9dfb571b298b04504e7a)).
Unstaging a file and literally remove a file are two very different things.

**Pull request closed without merging**